### ISSUE TITLE
Conversion of Stoa0058

### ISSUE THREAD

**Issue by PonteIneptique** [PerseusDL/canonical-latinLit#88]
So the conversion of Boetius will be easy. 
...
@ThomasK81 @lcerrato @AlisonBabeu @annettegessner Through, the way the text are quoted right now is probably not what we want on the long run (?), *ie* is LoebLine the best citation system for a text.

I'll convert them ignoring loebline for now but I'd be happy to have your feeling about that

**Comment by ThomasK81** [comment]
@PonteIneptique I agree. LoebLine is not a good way to cite Boethius. We need a logical citation. He is part of PL anyway.

**Comment by PonteIneptique** [comment]
So what should I convert it to ?

**Comment by ThomasK81** [comment]
Since the divs itself are not the problem, I'd would break on the sentence level in the prose part until we have a better citation system for the text.

**Comment by lcerrato** [comment]
can we be sure to preserve some of the current numbering system, even if commented out? otherwise editing across versions becomes difficult as content searching is not always practical.
https://github.com/PerseusDL/canonical/issues/156

**Comment by PonteIneptique** [comment]
It is not actually div but milestone, which means that whatever we do, we will keep this informations.

**Comment by PonteIneptique** [comment]
Here is an example : https://github.com/PerseusDL/canonical-latinLit/blob/stoa0058/data/stoa0058/stoa028/stoa0058.stoa028.perseus-lat1.xml

**Comment by PonteIneptique** [comment]
I would actually assume that div every ~20 lines, splitting on a dot, would probably be a good first step...
